packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -674,3 +676,145 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'}];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+function Copyable() {
+  let {clipboardProps} = useClipboard({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'
+      }];
+    }
+  });
+
+  return (
+    <div role="textbox" tabIndex={0} {...clipboardProps}>
+      Copy me 
+    </div>
+  );
+}
+
+export const DropzoneExampleWithInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropZone Area
+      <Input />
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithLinkAndInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Link slot="file">Upload</Link>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithButton = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Button slot="file">upload</Button>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithDraggableObject = (props) => (
+  <div>
+    <Draggable />
```

Comment:
this example is a bit odd to use. I tab to the Draggable, hit enter, but then from what I can tell, focus doesn't move to the dropzone. @reidbarber and I are happy to help you debug this if you need it